types/datadog-winston/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for datadog-winston 1.0
+// Project: https://https://github.com/itsfadnis/datadog-winston
+// Definitions by: Matt Hintzke <https://github.com/mhintzke13>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+import * as Transport from "winston-transport";
+
+export interface DataDogTransportOptions extends Transport.TransportStreamOptions {
+    apiKey: string;
+    hostname: string;
+    service: string;
+    ddsource: string;
+    ddtags: string;
+}
+
+export class DataDogTransport extends Transport {
```

Comment:
Instead of exporting declaration you want to write `export = DataDogTransport` since the usage of this is via `const x = require("DataDogTransport")` as per https://github.com/itsfadnis/datadog-winston#usage